src/main/java/org/terasology/manualLabor/components/ShearableComponent.java

```diff
@@ -0,0 +1,22 @@
+// Copyright 2021 The Terasology Foundation
+// SPDX-License-Identifier: Apache-2.0
+
+package org.terasology.manualLabor.components;
+
+import org.terasology.engine.entitySystem.Component;
+
+/**
+ * Attached to components that can be sheared.
```

Comment:
Components are not attached to components.
This sounds like this component is automagically attached, but from a developer's perspective this rather is a component that _can_ be attached.